tester_app/client/app/bundles/HelloWorld/startup/helloWorldClientEntryComponent.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
@alexfedoseev you said to do it this way, though, right? And I know this is the non-server-rendering version, but with the server-rendering version you need two files because you may want to initialize server code differently than client code, and this is where that would go?